charts/pulsar/templates/broker/broker-configmap.yaml

```diff
@@ -78,6 +78,14 @@ data:
   functionsWorkerEnabled: "false"
   {{- end }}
 
+  {{- if and .Values.components.functions .Values.functions.functionMesh }}
+  # enable package service for function mesh
+  enablePackagesManagement: "true"
+  packagesManagementStorageProvider: org.apache.pulsar.packages.management.storage.bookkeeper.BookKeeperPackagesStorageProvider
+  packagesReplicas: "1"
```

Comment:
should set to `{{ .Values.broker.configData.managedLedgerDefaultEnsembleSize }}`, which same as functions worker's `numFunctionPackageReplicas`